src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+import { findSelectedTab } from '../../helpers';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = findSelectedTab(tabs, selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+            const isTabSelected = id === selectedTab.id;
+
+            return (
+              <li
+                className={classNames({
+                  'is-active': isTabSelected,
+                })}
+                data-cy="Tab"
+                key={id}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (!isTabSelected) {
+                      onTabSelected(tab);
```

Comment:
i think your code will be look more simplify if you pass `id` in the `onTabSelected` function `const onTabSelected = tabId => setSelectedTabId(tabId);`